inspirehep/modules/converttohal/tei.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+from jinja2 import Environment, PackageLoader
+import json
+
+TEMPLATE = 'default.xml'
+#from inspirehep.utils.record_getter import get_es_record
+from inspirehep.modules.records.json_ref_loader import replace_refs
+#from inspirehep.modules.authors.utils import scan_author_string_for_phrases
+from nameparser import HumanName
+import sys
```

Comment:
PEP8 contains some suggestions on how to order the imports: https://www.python.org/dev/peps/pep-0008/#imports

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
>
> Imports should be grouped in the following order:
> 
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
>
> You should put a blank line between each group of imports.

Also, committing commented code should be avoided as much as possible (I'm almost sure you know this, but better safe than sorry).